.github/workflows/scripts/changelog-release-helper.test.mjs

```diff
@@ -54,6 +54,11 @@ describe('Changelog release helper', () => {
           badVersion: '3.0.2',
           type: 'patch',
           goodVersion: '3.0.1'
+        },
+        {
+          badVersion: '3.0.2-beta.0',
+          type: 'prepatch',
+          goodVersion: '3.0.1-beta.0'
```

Comment:
Should we explicitly test for what happens when we go from pre-release to release?

ie, the previous version is something like `3.0.1-beta.15` and the new version is `3.0.1`. That should pass.